fdm-core/src/cultivation.ts

```diff
@@ -240,13 +241,14 @@ export async function addCultivation(
 }
 
 /**
- * Retrieves the details of a specific cultivation.
+ * Retrieves details of a specific cultivation after verifying access permissions.
  *
- * @param fdm The FDM instance.
- * @param principal_id - The id of the principal that is requesting the cultivation
- * @param b_lu The ID of the cultivation.
+ * @param principal_id - The identifier of the principal requesting access.
```

Comment:
@coderabbitai Looks like the fdm argument is missing

Reply:
Ok, do you have maybe a better description of argument fdm? Then I will add that to all the other functions that are missing parameter description for argument `fdm`

Reply:
Mmw, what about: 
 `* @param fdm The FDM instance providing the connection to the database. The instance van be created with {@link createFdmServer}.`

Reply:
Yes

Reply:
Thanks, I have resolved in d348016ad77fdd0dfe95e2f22c4d4c537cb0e372